HLTrigger/special/python/hltppsJetComparisonFilter_cfi.py

```diff
@@ -0,0 +1,8 @@
+from Validation.CTPPS.hltppsJetComparisonFilterDefault_cfi import HLTPPSJetComparisonFilterDefault as _HLTPPSJetComparisonFilterDefault
```

Comment:
in any case `from Validation.CTPPS....` doens't look right if you are planning to making it derive from the `fillDescriptions`.

Reply:
You're right, fixing it rn

Reply:
this file has no reason to be, please remove it.